cas/incident-report.md

```diff
@@ -1,169 +1,384 @@
-5. The Lessons Learned section should contain the following subsections:
-   - What went well: a list of things that caused the incident to have less impact than it otherwise could have, such as early detection, rapid response, or good safety mechanisms. This section provides an opportunity for others to learn from the good practices of this CA Owner.
-   - What didn't go well: a list of things that caused the incident to have more impact than it otherwise would have, such as missing checks or unclear documentation. Each item here must have at least one corresponding Action Item below and should provide opportunities for others to ensure they make similar improvements if they haven't already.
-   - Where we got lucky: a list of things that went well, but which cannot be relied upon, such as early detection by an external security researcher or limited impact simply due to a small number of requests. Items here should generally also have corresponding Action Items, so that the CA Owner doesn't have to rely on luck in the future.
-6. The Action Items section must contain a list of remediation items that will be undertaken to ensure that similar incidents do not reoccur in the future. Note that it is not sufficient for these action items to simply stop this incident, they must create additional protections to prevent future incidents. Each Action Item should state:
-   - A short description of the action to be taken.
-   - A classification of whether the action will help _Prevent_ future incidents, _Mitigate_ the impact of future incidents, or _Detect_ future incidents. CA Owners are encouraged to propose action items in all three categories, with an emphasis on prevention and mitigation.
-   - A date by which the action item will be complete.
-7. The Appendix is for all supporting data: log files, graphs and charts, etc. In particular, in the case of incidents which directly impacted certificates, the Appendix must include a listing of the complete certificate details of all affected certificates. The recommended format is to ensure that all affected certificates are logged to CT, then to attach a text file where each line is of the form `https://crt.sh/?sha256=[sha256 fingerprint of the certificate]`. When the incident being reported involves an SMIME certificate, if disclosure of personally identifiable information in the certificate may be contrary to applicable law, please provide at least the certificate serial number and SHA256 hash of the certificate.
-
-### Incident Report Template
+In the case of Incident Reports with a Whiteboard field of "revocation-delay", Incident Reports MUST be updated every 72 hours to describe a summary of: 
```

Comment:
Frankly, this feels punitive. Is there much value in getting this info every 3 days rather than every 7?

Reply:
#187 updates this to...

"_In the case of Incident Reports with a Whiteboard field of "revocation-delay", reports SHOULD be updated every 3 days and MUST be updated no less frequently than every 7 days to describe a summary of..._"